### PR TITLE
allow to use customized pdf library

### PR DESCRIPTION
**Pull request by SailorMax**
This is:

- [ ] a bugfix
- [x] a new feature

Checklist:

- [ ] Changes are covered by unit tests
- [x] Code style is respected
- [x] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] CHANGELOG.md contains a short summary of the change
- [ ] Documentation is updated as necessary

What does it change?
  Allow to use customized PDF library without rewriting save() method.
  Customized PDF libray sample: https://tcpdf.org/examples/example_003/

### CODE REVIEW

File: src/PhpSpreadsheet/Writer/Pdf/Dompdf.php [hunk 1]
```diff
@@ -9,6 +9,18 @@
 class Dompdf extends Pdf implements IWriter
 {
     /**
+     * Gets the implementation of external PDF library that should be used.
+     *
+     * @param array $config Configuration array
```
Comment (PowerKiKi) [inline src/PhpSpreadsheet/Writer/Pdf/Dompdf.php]: There is no parameter named $config.

File: src/PhpSpreadsheet/Writer/Pdf/Dompdf.php [hunk 2]
```diff
@@ -9,6 +9,18 @@
...
+     *
+     * @return Dompdf implementation
```
Comment (PowerKiKi) [inline src/PhpSpreadsheet/Writer/Pdf/Dompdf.php]: This should be `\Dompdf\Dompdf`, not `Dompdf`

File: src/PhpSpreadsheet/Writer/Pdf/Mpdf.php [hunk 3]
```diff
@@ -10,6 +10,18 @@
 class Mpdf extends Pdf implements IWriter
 {
     /**
+     * Gets the implementation of external PDF library that should be used.
+     *
+     * @param array $config Configuration array
+     *
+     * @return Mpdf implementation
```
Comment (PowerKiKi) [inline src/PhpSpreadsheet/Writer/Pdf/Mpdf.php]: This should be `\Mpdf\Mpdf` not `Mpdf`

**Comment by SailorMax** [comment]
done.

But can you review my changes in docs/ and CHANGELOG.md? My English is not good for this kind of stuff :)

**Comment by PowerKiKi** [comment]
Thanks !

**Pull request closed without merging**